src/python/Utils/CertTools.py

```diff
@@ -1,4 +1,5 @@
 from __future__ import division, print_function
+from builtins import str
```

Comment:
It is not clear to me why we need it. Is it to keep the same byte string casting in py2 and py3, thus avoiding unicode strings in python3?

Reply:
I think futurize automatically added it since it saw that CertTools contained `str()`.

We have to decide what the function should return:

```
[...]
    key = cert = '/tmp/x509up_u' + str(os.getuid())
    if os.path.exists(key):
        return key, cert
```

Right now it is returning two bytes strings. Futurize made the assumption that we want to return two unicode strings and added `from builtins import str`. 

I suggest that even if we want to return bytes, we use `from builtins import bytes` and cast `os.getiud` to bytes, so that the string type will be uniform in python2 and python3.

Some more tests with unicode and bytes strings in py2 are available at https://cms-dmwm-test.docs.cern.ch/py2py3/cheatsheet/strings_future_py2/#unicode-bytes-basestring

Reply:
What if we change it from:
```
key = cert = '/tmp/x509up_u' + str(os.getuid())
```
to 
```
key = cert = '/tmp/x509up_u{}'.format(os.getuid())
```
?